.azure-pipelines/release-dev.yml

```diff
@@ -0,0 +1,42 @@
+# Build the dev version of the package and publish to artifacts
+
+trigger: none
+
+pool:
+  vmImage: ubuntu-latest
+steps:
+  - displayName: 'Use Python 3.10'
+    task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.10'
```

Comment:
Not relevant to this PR: When shall we start to use Python 3.11 in onnxscript if there is no obvious blocker?

Reply:
I guess now?

Reply:
Done